torch/csrc/lazy/core/hash.h

```diff
@@ -181,6 +181,11 @@ static inline hash_t Hash(const hash_t& value) {
   return value;
 }
 
+template <typename T>
```

Comment:
can you also make a PR to land this part to master?

Reply:
Yea, for sure. Need more time to work on the upstream counterpart.